code/style.css

```diff
@@ -1,5 +1,172 @@
-/* If you can see a peach background, the css file is correctly linked. Feel free to change or remove this styling! */
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
 
 body {
-  background-color: peachpuff;
+  font-family: "Courier New", Courier, monospace;
+}
+
+header {
+  margin: 40px;
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+footer {
+  margin: 40px 40px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+#hamburger div {
+  width: 30px;
+  height: 5px;
+  background-color: black;
+  border-radius: 5px;
+  margin: 5px;
+  margin-left: 0;
```

Comment:
To be able to specify the different directions you can use this principle instead of having to write the same rule multiple times: 
`margin: top right bottom left;`
So to write these two rows using this principle, you'd write: 
`margin: 5px 5px 5px 0;`